README.md

```diff
@@ -1,2 +1,23 @@
 # personal-portfolio
 My portfolio of work-to-date
+
+<h1>Description:-</h1>
+First page created from scratch. A work in progress portfolio page of projects to come.
+Include links to my LinkedIn and Github pages. 
+I've also included a toolkit sidebar displaying tools used to date.
+Site is responsive for tablets and mobiles.
+
+<h2>Image of code:</h2>
+<img src=".\assets\images\code-screenshot.png"/>
+
+<h2>Images of website:</h2>
+<img src=".\assets\images\banner-screenshot.png"/>
+<img src=".\assets\images\container-screenshot.png"/>
+
+<h3>Link to deployed page:</h3>
```

Comment:
Consider moving the deployed link nearer to the top of the README

Reply:
I'll do that. Thank you :)